bin/add-services.sh

```diff
@@ -6,5 +6,5 @@ dir=$(dirname ${0})
 
 ${dir}/utils/idam-create-service.sh "ccd_gateway" "ccd_gateway" "ccd_gateway_secret" "http://localhost:3451/oauth2redirect"
 
-${dir}/utils/idam-create-service.sh "adoption" "adoption" "OOOOOOOOOOOOOOOO" "https://localhost:9000/oauth2/callback"
+${dir}/utils/idam-create-service.sh "adoption" "adoption" "adoption" "https://localhost:3000/landing"
```

Comment:
Ok so I am nervous to see /landing here, it should allow any page commng from https://localhost:3000 not only landing one. Trying to reach out to idam team if it's supported

Reply:
For now merging this to master, once we get confirmation can decide on the next step

Reply:
@akomisarek we are not going to add anymore redirects here... its just 1. I explained this in my zoom session. I will show it to you when you have couple of mins.